modules/account-lib/src/coin/near/transactionBuilder.ts

```diff
@@ -140,29 +144,75 @@ export abstract class TransactionBuilder extends BaseTransactionBuilder {
 
   // endregion
 
-  public sender(value: string) {
-    this._sender = value;
+  /**
+   * Sets the public key and the address of the sender of this transaction.
+   *
+   * @param {SenderParams} publicKey the public key and account that is sending this transaction
+   * @returns {TransactionBuilder} This transaction builder
+   */
+  public sender({ address, pubKey }: SenderParams): this {
```

Comment:
public sender(address: string, pubKey: string) seems simpler

Reply:
from what I saw it,  always try to use  an interface for the parameters, but if you like i change it

Reply:
lets rollback and use two parameters.